core/templates/dev/head/domain/topic_viewer/ReadOnlyTopicObjectFactorySpec.ts

```diff
@@ -0,0 +1,116 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Tests for ReadOnlyTopicObjectFactory.
+   */
+
+import { TestBed } from '@angular/core/testing';
+
+import { ReadOnlyTopic, ReadOnlyTopicObjectFactory } from
+  'domain/topic_viewer/ReadOnlyTopicObjectFactory';
+
+import { SubtopicObjectFactory } from 'domain/topic/SubtopicObjectFactory';
+
```

Comment:
Delete unwanted newlines. (Check the guidelines [here](https://github.com/oppia/oppia/wiki/Coding-style-guide#webpack))